rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -333,6 +334,17 @@ def __init__(
 
         self.split_entities_config = self.init_split_entities()
 
+        self.finetune_mode = finetune_mode
+
+        if not self.model and self.finetune_mode:
+            raise rasa.utils.exceptions.RasaException(
```

Comment:
would be great if we could use a more specific sub class (or create one) instead of throwing this very generic exception type.

Reply:
Would `InvalidParameterException` be better?